modules/dnn/src/onnx/onnx_importer.cpp

```diff
@@ -1883,6 +1884,45 @@ void ONNXImporter::parseInstanceNormalization(LayerParams& layerParams, const op
     }
 }
 
+void ONNXImporter::parseGroupNormalization(LayerParams& layerParams, const opencv_onnx::NodeProto& node_proto) {
```

Comment:
Is it possible to reuse `parseInstanceNormalization`? This funcion looks the same as `parseInstanceNormalization`.